### PR TITLE
WIP LPS-158500 [Raylife AP] Create Review Quote Table

### PR DESCRIPTION
**Pull request by aecioprado**

### CODE REVIEW

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+      <ClayTable borderless hover={false}>
+
+        <ClayTable.Head className='border-bottom'>
+          <ClayTable.Row className='no-gutters row'>
+            <ClayTable.Cell className='col-6'><span className='text-dark'><h6>COVERAGES</h6></span></ClayTable.Cell>
+
+            <ClayTable.Cell className='col-4'><span className='text-dark'><h6>PROTECTION AMOUNT</h6></span></ClayTable.Cell>
+
+            <ClayTable.Cell className='col'><span className='text-dark'><h6>PREMIUM</h6></span></ClayTable.Cell>
+          </ClayTable.Row>
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx:74]:
```suggestion
          <ClayTable.Row className='no-gutters row text-uppercase'>
            <ClayTable.Cell className='col-6'><span className='text-dark'><h6>Coverages</h6></span></ClayTable.Cell>

            <ClayTable.Cell className='col-4'><span className='text-dark'><h6>Protection Amount</h6></span></ClayTable.Cell>

            <ClayTable.Cell className='col'><span className='text-dark'><h6>Premium</h6></span></ClayTable.Cell>
          </ClayTable.Row>
```

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+import {NewApplicationAutoContext} from '../../../context/NewApplicationAutoContextProvider';
+const Review = () => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [state] = useContext(NewApplicationAutoContext);
+  const data = [
+    {
+      COVERAGE: 'BODILY INJURY',
+      PROTECTIONAMOUNT: '$0.00 / $0.00',
+      PREMIUM: '$0.00 ',
+    },
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx:28]: It's a good practice to write the object keys in lowercase :)

```suggestion
    {
      coverage: 'BODILY INJURY',
      protectionAmount: '$0.00 / $0.00',
      premium: '$0.00 ',
    },
```

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+                  </ClayTable.Cell>
+                </ClayTable.Row>
+              </>
+            );
+          })}
+        </ClayTable.Body>
+
+        <ClayTable.Head className='border-top'>
+          <ClayTable.Row className='no-gutters row'>
+            <ClayTable.Cell className='col-6' headingCell><span className='text-dark'><h6>TOTAL PREMIUM</h6></span></ClayTable.Cell>
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx:101]:
```suggestion
            <ClayTable.Cell className='col-6 text-uppercase' headingCell><span className='text-dark'><h6>Total Premium</h6></span></ClayTable.Cell>
```

**Review by joanagrego — CHANGES_REQUESTED** [review]
Nice job! I added some suggestions :)

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx (hunk 2 above)
Comment (aecioprado) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx:28]: thanks

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx (hunk 3 above)
Comment (aecioprado) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx:101]: thanks

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayTable from '@clayui/table';
+import React, {useContext} from 'react';
+
+import {NewApplicationAutoContext} from '../../../context/NewApplicationAutoContextProvider';
+const Review = () => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx:21]: I believe that you need to delete this line.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx [hunk 5]
```diff
@@ -0,0 +1,113 @@
+/* eslint-disable sort-keys */
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx:1]: I believe that you need to delete this line.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+import {NewApplicationAutoContext} from '../../../context/NewApplicationAutoContextProvider';
+const Review = () => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [state] = useContext(NewApplicationAutoContext);
+  const data = [
+    {
+      coverage: 'BODILY INJURY',
+      protectionAmount: '$0.00 / $0.00',
+      premium: '$0.00 ',
+    },
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx:28]: Object keys should be in natural ascending order. The key 'premium' should be before 'protectionAmount'.
```suggestion
    {
      coverage: 'BODILY INJURY',
      premium: '$0.00 ',
      protectionAmount: '$0.00 / $0.00',
    },
```

**Review by joanagrego — CHANGES_REQUESTED** [review]
Nice job! I added some suggestions :)

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx (hunk 5 above)
Comment (aecioprado) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx]: done. thanks

**Review by joanagrego — APPROVED** [review]
LGTM! Thanks @aecioprado!

**Comment by joanagrego** [comment]
ci:test:sf

**Comment by joanagrego** [comment]
ci:test:sf

**Comment by aecioprado** [comment]
ci:test:sf

**Comment by aecioprado** [comment]
ci:test:sf

**Comment by joanagrego** [comment]
ci:test:sf

**Comment by joanagrego** [comment]
ci:test:sf

**Comment by aecioprado** [comment]
ci:test:sf

**Comment by aecioprado** [comment]
ci:test:sf

**Comment by joanagrego** [comment]
ci:test:sf

**Pull request closed without merging**